BlueprintUI/Sources/Layout/GridRow.swift

```diff
@@ -0,0 +1,231 @@
+import UIKit
+
+/// Like `Row`, `GridRow` displays a list of items in a linear horizontal layout. Unlike `Row`, `GridRow` provides
+/// convenience for describing columnar layout.
+///
+/// Horizontally, `GridRow` children are stretched to fill the available space. Vertically, children are aligned
+/// according to the `verticalAlignment` property.
+///
+/// Children may be sized proportionally or absolutely. Proportionally-sized children are granted a proportion of
+/// the total layout space **after** absolutely-sized children and margins have been subtracted.
+///
+/// ## Example:
+///
+/// ```
+/// GridRow { row in
+///   row.verticalAlignment = .fill
+///   row.spacing = 8.0
+///   row.add(child: .proportional(name, 0.75))
+///   row.add(child: .proportional(number, 0.25))
+///   row.add(child: .absolute(status, 50))
+/// }
+/// ```
+///
+/// ## Expected layout:
+///
+/// ```
+/// ┌────────────────────────────┬─┬────────┬─┬──────────────────┐
+/// │            name            │ │ number │ │      status      │
+/// │            (75%)           │ │  (25%) │ │     (50 pts)     │
+/// │                            │ │        │ │                  │
+/// ●──────────── 75  ───────────● ●── 25 ──● ●─────── 50 ───────●
+/// └────────────────────────────┴─┴────────┴─┴──────────────────┘
+/// ●──────────────────────────── 166 ───────────────────────────●
```

Comment:
Should this value be 150?

Reply:
It might be helpful to use a value where the proportional parts don't add up to 100 pts to make it more clear how space is distributed.

Reply:
It's subtle, but the extra 16 is attributed to spacing. I'll refine this diagram for clarity.

Reply:
Ah, missed the spacing—makes sense!

Reply:
Cleaned this up a bit by adding numbers to better illuminate spacing and doubling the other figures used.